### DISCUSSION TITLE
Observing ConstraintViolationException during org-formation init-pipeline execution run.

### DISCUSSION THREAD

**Discussion by KubeKD16** [org-formation/org-formation-cli#331] [General]
I recently started using org-formation for my organization and while I was setting up the org-formation pipeline, during the first run post pipeline creation, the build fails with the below error - 

`

ConstraintViolationException: You have exceeded the maximum policy size.
--
46 | ERROR: Task OrganizationUpdate execute failed. reason: You have exceeded the maximum policy size.
47 | You have exceeded the maximum policy size. (use option --print-stack to print stack)

`

Did anyone face the similar issue? Is this something to be worried about as it's my first initial run for the pipeline, so not sure if this is an expected behavior? The organization that I setup is fairly new, so is this a limitation that AWS imposes on newly created organization account & if there is any scope for the Policy size to be increase? 

Any thoughts/suggestions on this are hugely appreciated. 

Thanks
![Screen Shot 2022-03-17 at 11 46 28 AM](https://user-images.githubusercontent.com/79280270/158838900-6b216dde-646f-4d5d-9d0b-0f3de5a2ce3e.png)

**Comment by KubeKD16** [comment]
Re-running the CodeBuild project worked successfully. Kindly ignore/archive this discussion.

**Comment by OlafConijn** [comment]
this is an error that was returned from the organizations service.
